inc/xeve.h

```diff
@@ -592,6 +592,7 @@ int  XEVE_EXPORT xeve_encode(XEVE id, XEVE_BITB * bitb, XEVE_STAT * stat);
 int  XEVE_EXPORT xeve_config(XEVE id, int cfg, void * buf, int * size);
 int  XEVE_EXPORT xeve_param_default(XEVE_PARAM* param);
 int  XEVE_EXPORT xeve_param_ppt(XEVE_PARAM* param, int profile, int preset, int tune);
+int  XEVE_EXPORT xeve_check_conf(XEVE_CDSC* cdsc);
```

Comment:
I'd like to suggest the followings;
- change the "xeve_check_conf()" API name to "xeve_param_check()"
- use XEVE_PARAM* param instead of XEVE_CDSC* cdsc for input argument, because it would be kinds of utility function for parameter setting.

Reply:
Done